### ISSUE TITLE
ATD Persona 1 Recruitment Strategy

### ISSUE THREAD

**Issue by kim-tellez** [hackforla/access-the-data#147]
### Overview
We need to develop a recruitment strategy that targets finding Community Activists. 

### Action Items

- [x] Create recruitment strategy with sample email & LinkedIn messaging templates
- [x] Create contacts spreadsheet
- [x] Create sample script/list of questions for Informational Interviews
- [x] Create Screener Survey Questions (in Google Doc to allow for comments)
- [x] Create Screener Survey (Google Form)
- [x] Review with PMs 

### Resources/Instructions
[Recruitment Strategy Doc](https://docs.google.com/document/d/1AJtclpyQv8PLpr00IbZFZdXcSCCf4VbdHsh3JR6oEDU/edit?usp=sharing )

[Contacts Spreadsheet](https://docs.google.com/spreadsheets/d/1lE-0M2Fc9Njf71aG9wrlE5gXhyWh_hV1AnXEtKnu558/edit?usp=sharing)

[Sample script/list of questions for Informational Interviews](https://docs.google.com/document/d/1zDYKc1m3r5GwK3jHmF6V_vq8GpqX-hmp1mfNk3fCuYc/edit?usp=sharing)

[Screener Survey Questions](https://docs.google.com/document/d/1PTEBRiPYWQarMyXMwCxGZy_OqW4qu5MSGfqCX7kZSxM/edit?usp=sharing)

[Screener Survey](https://docs.google.com/forms/d/e/1FAIpQLSd-lHSN9DLLzvpCtm05zKGkbHRQ2LoIvbZjJ9ilOcgV0-Xeqg/viewform?usp=sf_link)

[Screener Survey Ready to Send](https://forms.gle/8zA4j8tqwx4LceDX8)

[email signatures](https://docs.google.com/document/u/3/d/1DYj5weYdvgz_PqsnpLM-WFQdED5YyPXDkWKmgEuXvY0/edit)

Wiki 
https://github.com/hackforla/access-the-data/issues/179

**Comment by ExperimentsInHonesty** [comment]
@kimberly-tellez The email is great.  Please feel free to reach out to organizations other than Neighborhood Councils with this message.  If you have neighborhood councils on your list we need to coordinate them with our department stakeholder (the Department of Neighborhood Councils).

**Comment by lrchang2** [comment]
@kimberly-tellez Please add update using this template (even if you have a pull request)

Progress: "What is the current status of your project? What have you completed and what is left to do?"
Blockers: "Difficulties or errors encountered."
Availability: "How much time will you have this week to work on this issue?"
ETA: "When do you expect this issue to be completed?"
Pictures: "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 

1. ask for help at your next meeting
2. put a "Status: Help Wanted" label on your issue and pull request
3. put up a request for assistance on the #access-the-data channel.

**Comment by lrchang2** [comment]
look into mail merge tool (through hackforla email) as seen used in the [internship project](https://github.com/hackforla/internship/issues/129) to automate and customize emails

**Comment by lrchang2** [comment]
@kimberly-tellez Please add update using this template (even if you have a pull request)

Progress: "What is the current status of your project? What have you completed and what is left to do?"
Blockers: "Difficulties or errors encountered."
Availability: "How much time will you have this week to work on this issue?"
ETA: "When do you expect this issue to be completed?"
Pictures: "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either: 

1. ask for help at your next meeting
2. put a "Status: Help Wanted" label on your issue and pull request
3. put up a request for assistance on the #access-the-data channel.

**Comment by Aparna1Gopal** [comment]
**Notes**
Was paired with Kim-Tellez to work on this issue by Lucy Chang, PM as a good first issue post Onboarding on April 19, 2022. Was introduced to Kim-Tellez by PM Lucy Chang via Slack. Will be meeting with all the UXRs currently working on the User Personas.

**Comment by Aparna1Gopal** [comment]
UXRs meeting to discuss all User Personas on ATD. 
https://github.com/hackforla/access-the-data/issues/154#issuecomment-1105862512

**Comment by Aparna1Gopal** [comment]
Progress: "What is the current status of your project? What have you completed and what is left to do?"
Currently reviewing the Screener Survey before it is sent out to research participants 
Working on expanding the research participant database
The item list currently being worked on is mentioned here: https://github.com/hackforla/access-the-data/issues/147#issuecomment-1106074562

- [ ] First review of Screener Survey 
- [ ] Second review of Screener Survey 
- [ ] Final Screener Survey ready to send out to research participants
- [ ] 

Blockers: "Difficulties or errors encountered."
I only got onboarded recently, so navigating across multiple pages to try and understand the requirements/need/progress of project. This ties in directly to the needs of the Screener Survey (example: why are certain questions on the survey, is it really necessary etc). Planning to set up call with Kim Tellez to discuss this further and get an better overview. 

Availability: "How much time will you have this week to work on this issue?"
6 hours including meetings 

ETA: "When do you expect this issue to be completed?"
By next Tuesday's meeting 

Pictures: "Add any pictures of the visual changes made to the site so far."

If you need help, be sure to either:

ask for help at your next meeting
put a "Status: Help Wanted" label on your issue and pull request
put up a request for assistance on the #access-the-data channel.

**Comment by Aparna1Gopal** [comment]
Progress: "What is the current status of your project? What have you completed and what is left to do?"
Screener survey, my review and first draft of Screener complete. Workshopped with Kim to work on Draft 2 and finalize it ready for circulation

Blockers: "Difficulties or errors encountered."
Understanding the foundational needs of the screener. Will be collaborating and workshopping with Kim on May 04, 2022 to work around this. 
Collaborated with Kim T for workshop. However, there were other blockers that arose that are discussed in further 

Availability: "How much time will you have this week to work on this issue?"
6 hours (excluding meetings) 

ETA: "When do you expect this issue to be completed?"
Current issue regarding Screener Survey is complete
The screener has been reviewed by Kim T and next by Aparna G. Needs PM’s review.  See notes 

Pictures: "Add any pictures of the visual changes made to the site so far."
Screener Survey 
If you need help, be sure to either:
1.	ask for help at your next meeting
2.	put a "Status: Help Wanted" label on your issue and pull request
3.	put up a request for assistance on the #access-the-data channel.

**Comment by Aparna1Gopal** [comment]
**Notes: 

Aparna G and Kim T met on May 04, 2022 5PM PST to discuss/workshop the Screener Survey** 

Is the Screener Survey still applicable? Since the recruitment strategy now includes reaching out to potential research participant via LinkedIn/Email, then having an informational interview with them. Initially Screener Survey was to be sent after this stage. But since the informational interview already provided the basics regarding the user, is the screener necessary now? 

Discussed below are various Options that the screener could be used for :
Option 1 could be that the screener could be sent as a first point of contact along with the ATD’s introduction slide deck to potential research participants.

Option 2, the research participant who has already completed an informational interview with ATD can send this Screener Survey to their network (to recruit additional research participants) 

Option 3, Since all the groundwork has been completed on the Screener this resource can be submitted as a blog/guide to Access the Data’s Toolkit. Will be reviewing with PM and UX Lead. 
(Aparna G and Kim T decided to work on a design sprint format to further develop Option 3. Will be following up with PM and UX Lead before commencing this sprint) 
Discuss hosting the Screener Survey blog/toolkit (Where will the Screener write-up be hosted – will it be on the toolkit page, GitHub Guides etc.) 


Aparna G will continue pairing with Kim T to work on this issue #147 while also working on Aparna G’s individual issue.

**Comment by Aparna1Gopal** [comment]
**Notes**

@Aparna1Gopal Tasks

- [ ] Expanding the research participant database. Researching and updating recruitment database.

https://docs.google.com/spreadsheets/d/1lE-0M2Fc9Njf71aG9wrlE5gXhyWh_hV1AnXEtKnu558/edit#gid=0

[Currently on hold? Based on last week’s ATD Meeting pertaining to citizen engagement, wherein it was mentioned a collective outreach would be preferable instead of individual outreach from team, based on stakeholder requirements. Need to follow up with PM and UX Lead] 

- [ ] Final Screener Survey ready to send out to research participants

[On hold, until the further discussions with PM, See above notes]

**Comment by bradyse** [comment]
**Notes**

After ATD Research breakout meeting on Wed 5/11, we determined that @joyceeyang needs to send out this survey to our most recent community activist contact (Chris). Once @Aparna1Gopal okays the edits from the Final Screener Survey, @bradyse will make the edits in the actual Google Form and send the link to Joyce to send out.

**Comment by Aparna1Gopal** [comment]
Progress: "What is the current status of your project? What have you completed and what is left to do?"
Screener Survey complete 

Blockers: "Difficulties or errors encountered."
I had a blocker since I was not sure which direction to proceed with the Screener. Since this was a paired issue, there are multiple stakeholders required to take this decision. 
I have Slack communicated with Lucy C, PM and Sara B, UXR (currently working on the Google Form) and cleared this blocker. 

Availability: "How much time will you have this week to work on this issue?"
2 hours 

ETA: "When do you expect this issue to be completed?"
The Screener has been completed. 
 
Pictures: "Add any pictures of the visual changes made to the site so far."
https://docs.google.com/document/d/1PTEBRiPYWQarMyXMwCxGZy_OqW4qu5MSGfqCX7kZSxM/edit

If you need help, be sure to either:

ask for help at your next meeting
put a "Status: Help Wanted" label on your issue and pull request
put up a request for assistance on the #access-the-data channel.

**Comment by Aparna1Gopal** [comment]
@bradyse Please find the Final Screener Survey here https://docs.google.com/document/d/1PTEBRiPYWQarMyXMwCxGZy_OqW4qu5MSGfqCX7kZSxM/edit

@joyceeyang @lrchang2

**Comment by bradyse** [comment]
> 

Made edits, had one more question about the required responding regarding the "General Questions" section. I asked Aparna in Slack about it. Once that is confirmed, then the survey is ready to go.

**Comment by Aparna1Gopal** [comment]
@bradyse Noted and Done!

**Comment by bradyse** [comment]
@joyceeyang Here is the link to the Screener Survey to send to Chris: https://forms.gle/8zA4j8tqwx4LceDX8

Thank you @Aparna1Gopal for your help!!

**Comment by joyceeyang** [comment]
Got it! Thank you @bradyse and @Aparna1Gopal

**Comment by LouismChan** [comment]
Hi Joyce (@joyceeyang ) This is the link for data experts: https://forms.gle/3ofdUikFkbBK5Vbf6.  Please send this link for data experts to Chris also.  Thanks.

**Comment by LouismChan** [comment]
Thanks Joyce (@joyceeyang ), Chris Ringewald has filled out the Data Expert survey.
